pkg/reaper/nodereaper/nodereaper_test.go

```diff
@@ -40,6 +40,15 @@ import (
 
 var loggingEnabled bool
 
+type SkipLabel string
```

Comment:
any specific reason we need a dedicated type and constants for this? vs. using the existing constants in actual code?

Reply:
I originally meant to use this type as an enum to restrict the kind of skip labels there were, but I ended up scrapping that idea but kept this type. I can convert it to just use the existing constants.